### PR TITLE
Update calculation for the historic ASFR

### PR DESCRIPTION
**Pull request by Eleonore9**
- New data input for testing purpose (ONS projections of births by age of mother)
- New data output from R for testing purpose (historic ASFR for Bristol)
- New schema for the new data input
- Remove all mention of the "Base ASFR"
- Update/add to the fertility module (`fertility_mvp.clj`)

### CODE REVIEW

File: src/witan/models/dem/ccm/fert/fertility_mvp.clj [hunk 1]
```diff
@@ -9,20 +9,31 @@
...
+                   {:year (dec base-year)}))
+
+(defn calc-asfr-birth-data-yr
+  "Takes the population at risk for the birth data year and ONS
+  projections for births by age of mother. Returns fertility
+  rates for the birth data year."
+  [popn-at-risk-birth-data-yr ons-proj-births-by-age-mother]
+  (-> ons-proj-births-by-age-mother
+      (wds/join popn-at-risk-birth-data-yr [:gss-code :sex :age])
+      (wds/add-derived-column :births [:births] (fn [b] (if (nil? b) 0.0 b)))
```
Comment (acron0) [inline src/witan/models/dem/ccm/fert/fertility_mvp.clj]: `(or b 0.0)`

File: src/witan/models/dem/ccm/fert/hist_asfr_ws.clj [hunk 2]
```diff
@@ -5,9 +5,9 @@
             [witan.models.dem.ccm.schemas :refer :all]))
 
 ;; GET DATA FOR ROOT NODES
-(def root-data-paths {:births-data "resources/test_data/bristol_births_data.csv"
-                      :at-risk-popn "resources/test_data/bristol_denominators.csv"
-                      :hist-births-est "resources/test_data/bristol_hist_births_est.csv"})
+(def root-data-paths {:births-data "test_data/bristol_births_data.csv"
```
Comment (acron0) [inline src/witan/models/dem/ccm/fert/hist_asfr_ws.clj]: What are these paths used for outside of testing?

Comment (Eleonore9) [inline src/witan/models/dem/ccm/fert/hist_asfr_ws.clj]: @acron0 they're only used in this namespace (`hist_asfr_ws.clj`) where we tried to implement the workflow before and in the test namespaces.

Comment (acron0) [inline src/witan/models/dem/ccm/fert/hist_asfr_ws.clj]: I'd kill this whole file, plus your dependency on workspace-executor

Comment (Eleonore9) [inline src/witan/models/dem/ccm/fert/hist_asfr_ws.clj]: oh sure! We somehow kept it just in case.

**Comment by acron0** [comment]
:shipit:

**Comment by tcoupland** [comment]
:shipit:

**Pull request merged**